.ci/Dockerfile

```diff
@@ -17,12 +17,15 @@ ENV HELM_VERSION=3.9.1
 RUN curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
     | sh -s -- -b $(go env GOPATH)/bin v${GOLANGCILINT_VERSION}
 
-# kubebuilder to get required tools (etcd, apiserver, etc.)
-ENV PATH=${PATH}:/usr/local/kubebuilder/bin
-RUN curl -fsSLO https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${KUBEBUILDER_VERSION}/kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64.tar.gz && \
-    tar -zxf kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64.tar.gz && \
-    mv kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64 /usr/local/kubebuilder && \
-    rm kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64.tar.gz
+# etcd and apiserver for integration tests
+ENV KUBEAPISERVER_VERSION=1.21.14
```

Comment:
Do we want to keep all of these at the top of the file together?

Reply:
It was a bad choice because it means that for any version change, we rebuild everything. I stopped doing this in the buildkite agent image. I think it was to have all the version grouped together. Anyway, this image should be removed soon.